Problem: upgrade to tendermint 0.18 fails

```
   Compiling tmkms-light v0.1.0 (/home/runner/work/tmkms-light/tmkms-light)
286
error[E0277]: the trait bound `tendermint::vote::SignVoteRequest: From<tendermint_proto::privval::SignVoteRequest>` is not satisfied
287
  --> src/rpc.rs:47:27
288
   |
289
47 |                 let svr = SignVoteRequest::try_from(req).map_err(|e| {
290
   |                           ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<tendermint_proto::privval::SignVoteRequest>` is not implemented for `tendermint::vote::SignVoteRequest`
291
   |
292
   = note: required because of the requirements on the impl of `Into<tendermint::vote::SignVoteRequest>` for `tendermint_proto::privval::SignVoteRequest`
293
   = note: required because of the requirements on the impl of `TryFrom<tendermint_proto::privval::SignVoteRequest>` for `tendermint::vote::SignVoteRequest`
294
   = note: required by `try_from`
295

296
error[E0277]: the trait bound `tendermint::proposal::SignProposalRequest: From<tendermint_proto::privval::SignProposalRequest>` is not satisfied
297
  --> src/rpc.rs:57:27
298
   |
299
57 |                 let spr = SignProposalRequest::try_from(spr).map_err(|e| {
300
   |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<tendermint_proto::privval::SignProposalRequest>` is not implemented for `tendermint::proposal::SignProposalRequest`
301
   |
302
   = note: required because of the requirements on the impl of `Into<tendermint::proposal::SignProposalRequest>` for `tendermint_proto::privval::SignProposalRequest`
303
   = note: required because of the requirements on the impl of `TryFrom<tendermint_proto::privval::SignProposalRequest>` for `tendermint::proposal::SignProposalRequest`
304
   = note: required by `try_from`
305

306
error[E0277]: the trait bound `tendermint::public_key::PubKeyRequest: From<tendermint_proto::privval::PubKeyRequest>` is not satisfied
307
  --> src/rpc.rs:67:27
308
   |
309
67 |                 let pkr = PubKeyRequest::try_from(pkr).map_err(|e| {
310
   |                           ^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<tendermint_proto::privval::PubKeyRequest>` is not implemented for `tendermint::public_key::PubKeyRequest`
311
   |
312
   = note: required because of the requirements on the impl of `Into<tendermint::public_key::PubKeyRequest>` for `tendermint_proto::privval::PubKeyRequest`
313
   = note: required because of the requirements on the impl of `TryFrom<tendermint_proto::privval::PubKeyRequest>` for `tendermint::public_key::PubKeyRequest`
314
   = note: required by `try_from`
315

316
error[E0277]: the trait bound `tendermint_proto::privval::SignedVoteResponse: From<tendermint::vote::SignedVoteResponse>` is not satisfied
317
   --> src/rpc.rs:162:67
318
    |
319
162 |             Response::SignedVote(resp) => Sum::SignedVoteResponse(resp.into()),
320
    |                                                                   ^^^^^^^^^^^ the trait `From<tendermint::vote::SignedVoteResponse>` is not implemented for `tendermint_proto::privval::SignedVoteResponse`
321
    |
322
    = help: the following implementations were found:
323
              <tendermint_proto::privval::SignedVoteResponse as From<tendermint::vote::sign_vote::SignedVoteResponse>>
324
    = note: required because of the requirements on the impl of `Into<tendermint_proto::privval::SignedVoteResponse>` for `tendermint::vote::SignedVoteResponse`
325

326
error[E0277]: the trait bound `tendermint_proto::privval::SignedProposalResponse: From<tendermint::proposal::SignedProposalResponse>` is not satisfied
327
   --> src/rpc.rs:163:75
328
    |
329
163 |             Response::SignedProposal(resp) => Sum::SignedProposalResponse(resp.into()),
330
    |                                                                           ^^^^^^^^^^^ the trait `From<tendermint::proposal::SignedProposalResponse>` is not implemented for `tendermint_proto::privval::SignedProposalResponse`
331
    |
332
    = help: the following implementations were found:
333
              <tendermint_proto::privval::SignedProposalResponse as From<tendermint::proposal::sign_proposal::SignedProposalResponse>>
334
    = note: required because of the requirements on the impl of `Into<tendermint_proto::privval::SignedProposalResponse>` for `tendermint::proposal::SignedProposalResponse`
335

336
error: aborting due to 5 previous errors
337

338
For more information about this error, try `rustc --explain E0277`.
```
https://github.com/crypto-com/tmkms-light/pull/14/checks?check_run_id=2119673471#step:5:330

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,12 +9,12 @@ edition = "2018"
 [dependencies]
 anomaly = "0.2"
 ed25519-dalek = "1"
-prost = "0.6"
+prost = "0.7"
 serde = { version = "1", features = ["serde_derive"] }
 subtle-encoding = { version = "0.5", features = ["bech32-preview"] }
-tendermint = { version = "0.17" }
-tendermint-proto = "0.17"
-tendermint-p2p = { version = "0.17" }
+tendermint = { version = "0.18" }
+tendermint-proto = "0.18"
+tendermint-p2p = { version = "0.18" }
 thiserror = "1"
 tracing = "0.1"
 

--- a/providers/nitro/nitro-enclave/Cargo.toml
+++ b/providers/nitro/nitro-enclave/Cargo.toml
@@ -11,8 +11,8 @@ ed25519-dalek = "1"
 nix = "0.17"
 serde_json = "1"
 subtle = "2"
-tendermint = { version = "0.17" }
-tendermint-p2p = { version = "0.17" }
+tendermint = { version = "0.18" }
+tendermint-p2p = { version = "0.18" }
 tmkms-light = { path = "../../.." }
 tmkms-nitro-helper = { path = "../nitro-helper", default-features = false }
 tracing = "0.1"

--- a/providers/nitro/nitro-helper/Cargo.toml
+++ b/providers/nitro/nitro-helper/Cargo.toml
@@ -25,7 +25,7 @@ structopt = "0.3"
 subtle-encoding = { version = "0.5", features = [ "bech32-preview" ] }
 sysinfo = { version = "0.16", optional = true }
 tempfile = "3"
-tendermint = { version = "0.17" }
+tendermint = { version = "0.18" }
 thiserror = "1"
 tmkms-light = { path = "../../.." }
 tokio = { version = "= 0.2", features = [ "rt-core", "time" ] }

--- a/providers/sgx/sgx-app/Cargo.toml
+++ b/providers/sgx/sgx-app/Cargo.toml
@@ -14,7 +14,7 @@ serde_json = "1"
 sgx-isa = { version = "0.3", features = ["sgxstd"] }
 subtle = "2"
 subtle-encoding = "0.5"
-tendermint-p2p = "0.17"
+tendermint-p2p = "0.18"
 tmkms-light-sgx-runner = { path = "../sgx-runner" }
 tmkms-light = { path = "../../.." }
 tracing = "0.1"

--- a/providers/sgx/sgx-runner/Cargo.toml
+++ b/providers/sgx/sgx-runner/Cargo.toml
@@ -10,7 +10,7 @@ ed25519 = { version = "1", features = ["serde"] }
 ed25519-dalek = "1"
 sgx-isa = "0.3"
 thiserror = "1"
-tendermint = { version = "0.17" }
+tendermint = { version = "0.18" }
 tmkms-light = { path = "../../.." }
 secrecy = { version = "0.7" }
 zeroize = "1"

--- a/providers/softsign/Cargo.toml
+++ b/providers/softsign/Cargo.toml
@@ -16,8 +16,8 @@ structopt = "0.3"
 subtle = "2"
 subtle-encoding = { version = "0.5", features = ["bech32-preview"] }
 tempfile = "3"
-tendermint = { version = "0.17" }
-tendermint-p2p = { version = "0.17" }
+tendermint = { version = "0.18" }
+tendermint-p2p = { version = "0.18" }
 tmkms-light = { path = "../.." }
 tracing = "0.1"
 tracing-subscriber = "0.2"

